include/sundials/sundials_adaptcontroller.h

```diff
@@ -145,6 +169,15 @@ SUNDIALS_EXPORT
 SUNErrCode SUNAdaptController_UpdateH(SUNAdaptController C, sunrealtype h,
                                       sunrealtype dsm);
 
+/* Function to notify the controller of a successful multirate time step
+   with size H and fast tolerance factor tolfac, and local error factors
+   DSM and dsm, indicating that the step size, tolerance factor, or local
+   error factors can be saved for subsequent controller functions. */
+SUNDIALS_EXPORT
+SUNErrCode SUNAdaptController_UpdateMRITol(SUNAdaptController C, sunrealtype H,
```

Comment:
Again, I do not see any fundamental flaw with the current name, particularly since this is only used internally within SUNDIALS.  Again, if others feel strongly then I'll consider changing it throughout the PR.

Reply:
Related discussion [here](https://github.com/LLNL/sundials/pull/564#discussion_r1805375991). Since this updates both `H` and `Tol` I though having both in the name would be more clear.